### DISCUSSION TITLE
Twitch Dev extensions review "paused"?

### DISCUSSION THREAD

**Discussion by kinnikinnick** [gameyfin/gameyfin#271] [General]
It seems this app cannot be used right now as it relies on Twitch Dev extensions and you can't sign up any new accounts currently as the "Reviews for Extensions, organizations, games, and chatbot verification are temporarily paused while we revise our processes. We are working to resume reviews as quickly as possible and will share updates with you shortly. Thank you for your patience and understanding."...

When you go to the 'register your application' page, as per the tutorial, after entering all the info when you click submit *nothing* happens. (I wish they had at least made a popup to explain what's going on)

Is there any way to set up gameyfin without a twitch developer account?

**Comment by kinnikinnick** [comment]
Omg, I did some digging around and the issue was simply that the form requires you to name the application with more than one word!  If you only put "gameyfin" clicking submit does nothing. If you name the app "gameyfin applications" it works as intended...  So dumb that they don't explain this at all in the applications page

**Reply by grimsi** [reply]
Glad you were able to resolve it.
Unfortunately Gameyfin v1 needs twitch credentials since it uses IGDB to load metadata for your games. Gameyfin v2 will have multiple plugins available for different metadata providers.
